app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter
+    post.increment!(:likes_count)
+  end
```

Comment:
- [OPTIONAL] As mentioned above you can use `after_save :update_likes_counter`(https://github.com/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/issues/4), after_save :update_comments_counter